hyperspy/tests/io/test_protochips.py

```diff
@@ -0,0 +1,287 @@
+# -*- coding: utf-8 -*-
+"""
```

Comment:
Could you replace this with the standard header?
(Sorry that I missed this before)